src/combat/BaseCombatTask.py

```
@@ -945,16 +1063,370 @@ def has_char(self, char_cls):
+            return True
+        self._team_binding_last_check = now
+
+        in_team, current_index, count = self.in_team()
+        if not in_team:
+            self._report_strict_team_error("队伍界面暂不可用, 已暂停旧队伍输入")
+            return False
+
+        manager = CustomCharManager()
+        visible = self._match_visible_team_preset(count, manager)
+        if visible is None:
+            # 阵亡后三人 HUD：已绑定深渊队且有角色死亡时，保留原队伍继续战斗，
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)